loglist/logfilter_test.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package loglist
+
+import (
+	"testing"
+
+	"github.com/google/certificate-transparency-go/testdata"
+	"github.com/google/certificate-transparency-go/x509"
+	"github.com/google/certificate-transparency-go/x509util"
+
+	"github.com/kylelemons/godebug/pretty"
+	"github.com/mohae/deepcopy"
+)
+
+func subLogList(logURLs map[string]bool) LogList {
+	var ll LogList
+	ll.Operators = deepcopy.Copy(sampleLogList.Operators).([]Operator)
```

Comment:
You're deep-copying `Operators` but shallow-copying `Logs`. Is there a reason for this? Make both shallow-copies if not.

Reply:
Done.